<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #8BC6E2;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="startLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 50px;
    font-weight: bold;
    color: orange;
}</string>
    </property>
    <property name="text">
     <string>Date span calculator</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#calendarWidget QWidget{
	alternate-background-color: orange;
	background-color: #5C7D8A;
}

#qt_calendar_navigationbar{


	border-top-left-radius: 5px solid black; 
	border-top-right-radius: 5px solid black;
}

#qt_calendar_prevmonth,
#qt_calendar_nextmonth{
	border: none;
	min-width: 13px;
	max-width: 13px;
	min-height: 13px;
	min-height: 13px;
	border-radius: 5px;
	background-color: transparent;
	padding:5px;
}


</string>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>from:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startingDate">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	background-color:orange;
	color:black;
}</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>till:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="endingDate">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	background-color:orange;
	color:black;
}
</string>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="date_select_button">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Narrow</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">    QPushButton {
        background-color: orange;
        color: black;
        border-radius: 15px;
        padding: 10px 20px;
        font-size: 15px;
        font-family: &quot;Arial Narrow&quot;, sans-serif;
    }
    
    QPushButton:pressed {
        background-color: darkorange; /* Change color when button is pressed */
    }</string>
       </property>
       <property name="text">
        <string>Select Date</string>
       </property>
       <property name="icon">
        <iconset theme="accessories-calculator">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="lostDays">
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Holiday Days:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberLostDays">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="working_days_label">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Working Days:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberWorkDays">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="days_selected_label">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Days Selected:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="DaysSelectedNumber">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>24</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:orange
}</string>
    </property>
    <property name="text">
     <string>Week</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>321</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="day_desc_title">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 24px;
    font-weight: bold;
    color: black;
}</string>
       </property>
       <property name="text">
        <string>Day Descritpion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="day_desc">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-family: Arial, sans-serif;
    font-size: 30px;
    font-weight: bold;
    color: orange;
}</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff9178;&quot;&gt;Lack of day info, please select date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculate_workflow_button">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">    QPushButton {
        background-color: orange;
        color: black;
        border-radius: 15px;
        padding: 10px 20px;
        font-size: 16px;
        font-family: &quot;Arial Narrow&quot;, sans-serif;
    }
    
    QPushButton:pressed {
        background-color: darkorange; /* Change color when button is pressed */
    }</string>
       </property>
       <property name="text">
        <string>Calculate Workflow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="optimize_workflow_button">
       <property name="styleSheet">
        <string notr="true">    QPushButton {
        background-color: orange;
        color: black;
        border-radius: 15px;
        padding: 10px 20px;
        font-size: 16px;
        font-family: &quot;Arial Narrow&quot;, sans-serif;
    }
    
    QPushButton:pressed {
        background-color: darkorange; /* Change color when button is pressed */
    }</string>
       </property>
       <property name="text">
        <string>Optimize Workflow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_button">
       <property name="styleSheet">
        <string notr="true">    QPushButton {
        background-color: orange;
        color: black;
        border-radius: 15px;
        padding: 10px 20px;
        font-size: 16px;
        font-family: &quot;Arial Narrow&quot;, sans-serif;
    }
    
    QPushButton:pressed {
        background-color: darkorange; /* Change color when button is pressed */
    }</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
